obspy/io/kinemetrics/evt_base.py

```diff
@@ -131,7 +131,7 @@ def _time(self, blocktime, param, val, offset):
             frame_milli = 0
         frame_time += 315532800  # diff between 1970/1/1 and 1980/1/1
         time = UTCDateTime(frame_time) + frame_milli / 1000.0
-        return UTCDateTime(ns=time.ns, precision=3)
```

Comment:
Why remove the specified precision? If the data format only stores miliseconds wouldn't we want this reflected in the ObsPy objects?

Reply:
Yep, you are correct. On the other side, a UTCDateTime object with non-default precision might become useless if you try to compare it to other UTCDateTime objects:
```
ObsPyDeprecationWarning: Comparing UTCDateTime objects of different precision is not defined will raise an Exception in a future version of obspy
```

I am OK with changing it back and filter the warning in the tests instead. But, this is the only place in ObsPy in which a different precision than the default is used.